diff --git a/schemas/meson.build b/schemas/meson.build
index 7e0b326..9d3f38a 100644
--- a/schemas/meson.build
+++ b/schemas/meson.build
@@ -3,7 +3,6 @@ gnome_extensions_theme_schemas = [
     'org.gnome.shell.extensions.dash-to-dock.gschema.override',
     'org.gnome.shell.extensions.desktop-icons.gschema.override',
     'org.gnome.shell.extensions.net.gfxmonk.impatience.gschema.override',
-    'org.gnome.shell.extensions.user-theme.gschema.override',
     'org.gnome.desktop.interface-theme.gschema.override',
     'org.gnome.desktop.wm.preferences.gschema.override',
     'org.gnome.shell-enabled-extensions.gschema.override',
diff --git a/schemas/org.gnome.desktop.interface-theme.gschema.override b/schemas/org.gnome.desktop.interface-theme.gschema.override
index 146f135..629f7a9 100644
--- a/schemas/org.gnome.desktop.interface-theme.gschema.override
+++ b/schemas/org.gnome.desktop.interface-theme.gschema.override
@@ -1,4 +1,2 @@
 [org.gnome.desktop.interface]
 font-name='Clear Sans 10'
-gtk-theme='Materia-compact'
-icon-theme='Paper'
diff --git a/schemas/org.gnome.desktop.wm.preferences.gschema.override b/schemas/org.gnome.desktop.wm.preferences.gschema.override
index cfd96cf..87148ba 100644
--- a/schemas/org.gnome.desktop.wm.preferences.gschema.override
+++ b/schemas/org.gnome.desktop.wm.preferences.gschema.override
@@ -1,4 +1,3 @@
 [org.gnome.desktop.wm.preferences]
 button-layout='appmenu:minimize,maximize,close'
-theme='Materia-compact'
 titlebar-font='Clear Sans Bold 11'
diff --git a/schemas/org.gnome.shell.extensions.user-theme.gschema.override b/schemas/org.gnome.shell.extensions.user-theme.gschema.override
deleted file mode 100644
index 70f3c68..0000000
--- a/schemas/org.gnome.shell.extensions.user-theme.gschema.override
+++ /dev/null
@@ -1,2 +0,0 @@
-[org.gnome.shell.extensions.user-theme]
-name='Materia-compact'
